.analysis_options

```diff
@@ -0,0 +1,29 @@
+analyzer:
+ strong-mode: true
+linter:
+  rules:
```

Comment:
Where does this list come from? Is this a published list, like from Effective Dart or something?